dev-docs/bidders/pgamssp.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: PGAMSSP
+description: Prebid PGAMSSP Bidder Adapter
+biddercode: pgamssp
+usp_supported: true
+tcfeu_supported: true
```

Comment:
If you claim `tcfeu_supported` you must also provide a `gvl_id`, otherwise this must be `false`

Reply:
thx, updated